### ISSUE TITLE
DeprecationWarning for xonsh completers

### ISSUE THREAD

**Issue by sergiopr89** [DangerOnTheRanger/xonsh-apt-tabcomplete#5]
Error shown:
```
/usr/local/lib/python3.6/dist-packages/xonsh/built_ins.py:1467: DeprecationWarning:
__xonsh_completers__ has been deprecated, please use __xonsh__.completers instead.
```
In order to fix it, change:
```
__xonsh_completers__["auto_apt"] = apt_completer
```
By:
```
__xonsh__.completers["auto_apt"] = apt_completer
```
In:
```
xontrib/apt_tabcomplete.xsh
```

**Comment by sergiopr89** [comment]
Hi @DangerOnTheRanger, when you have a moment, could you take a look and merge if proceeds, please?

Thank you.

**Comment by DangerOnTheRanger** [comment]
Closing, fixed by #6 - thanks again!
